README.md

```diff
@@ -1,10 +1,51 @@
 # the-crowd
-## Description :
 
+simple app that hosts ALL the ways an individual can take action on climate change and gives users an opportunity to ‘pledge’ to take that action.
 
 ## Heroku Link :
 
-
 ## Instructions :
+
+- clone the repository ,and cd to its directory
+- npm i to install packages in server package.json
+- npm i in client side
+- npm run dev to run server
+
 ## DB Schema :
+
 ![image](https://user-images.githubusercontent.com/27896127/66266898-d66db180-e833-11e9-952a-484f1c19e6af.png)
+
+## User Story :
+
+- As a user, the first page I see is the landing page so that I can I have an introduction what the app is.
+
+- As a user I have a home page so that I can view the pledges that I have already made (or be prompted to make my first pledge).
+
+- As a user, I can use an action dashboard so that I can navigate the various categories of pledges with ease.
+
+- As a user, I see up to three pledges within a category so that I have a vision of potential actions I can take within that category.
+
+- As as user I can find out more about a particular pledge so that I can decide if it is the right pledge for me.
+
+- As as user, I am reminded of the commitment that I am making so that I feel like my decision to make the pledge is important.
+
+- As a user, I am congratulated after I have confirmed to take the pledge so that I feel excited and supported in my decision.
+
+- As a user I can find out more information about the app so that I don’t feel confused.
+
+- As a user, my UX is consistent so that I trust the professionality of the brand, The Crowd.
+
+- As a user with a disability, I am able to navigate the app so that I do not feel discriminated against.
+
+- As a user, I can download this PWA onto my mobile device so that I can always have my pledges in my pocket.
+
+- As a user, I can always see my navbar at the bottom of my screen so that I can freely navigate the app.
```

Comment:
Don't put the user stories inside the readme, we've already put them in the issues.